content_scripts/hints.js

```diff
@@ -467,7 +489,12 @@ var Hints = (function() {
     }
 
     function createHints(cssSelector, attrs) {
```

Comment:
```suggestion
    function createHints(selector, attrs) {
```
We may want to rename the first argument to `createHints` since it can be either a CSS selector, RegExp, or Array.